agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java

```diff
@@ -166,6 +152,12 @@ private void generateFilter(BoolQueryBuilder bool, String filterName, String fil
 		//Log.info(bool);
 	}
 
+	private String escapeValue(String value) {
+		value = QueryParser.escape(value);
+		value = value.replaceAll("'", " ");
```

Comment:
@cmpich why not pull out the single quotes first then escape everything as the quotes will get escaped and this would leave the backslashes in the text string?